### DISCUSSION TITLE
How to launch and see app running?

### DISCUSSION THREAD

**Discussion by mdear** [File-New-Project/EarTrumpet#1649] [Q&A]
I'm running Win11 and installed EarTrumpet via the Windows Store.

When I access my sound system tray app, I don't see live sound bars running for my apps.  I am trying to identify an app that's being noisy. 

What do I need to do to evaluate EarTrumpet and find my noisy app?

![image](https://github.com/user-attachments/assets/443a085d-6aa5-40d3-9b13-6624b13d6a85)

![image](https://github.com/user-attachments/assets/fef0add7-726d-4d10-ac80-ac22008e2402)

![image](https://github.com/user-attachments/assets/ca27e60f-033c-4667-a3b8-4e5317691235)

![image](https://github.com/user-attachments/assets/83c5c0a3-fb8a-4d33-92ca-9f7bb6800a73)

Thanks!

Edition	Windows 11 Home
Version	24H2
Installed on	‎2024-‎11-‎17
OS build	26100.2605
Experience	Windows Feature Experience Pack 1000.26100.36.0

Processor	12th Gen Intel(R) Core(TM) i7-12700H   2.30 GHz
Installed RAM	64.0 GB (63.6 GB usable)
Device ID	458EAB2D-7D86-47D6-8E33-52E7738BF77E
Product ID	00342-21114-42277-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

**Comment by riverar** [comment]
Do you see EarTrumpet in the overflow area? This is the animation that plays at first start:
![welcome](https://github.com/user-attachments/assets/438e90e1-7464-43c6-8c01-aa40d6c7d119)
